lms/djangoapps/program_enrollments/models.py

```diff
@@ -18,12 +28,6 @@ class ProgramEnrollment(TimeStampedModel):  # pylint: disable=model-missing-unic
     .. pii_types: other
     .. pii_retirement: local_api
     """
-    STATUSES = (
```

Comment:
I think these are statuses only needed by Program Enrollment. 
The status for ProgramCourseEnrollment should just be "Active" vs "Inactive".
What was your understanding here?

Reply:
I thought that these were all also valid statuses for course enrollments, but it looks like according to the integration doc we will allow "Enrolled" , "Pending", and "Withdrawn"

Reply:
@kdmccormick and I made a change to it again yesterday. See https://docs.google.com/document/d/1o68unQ_ZEMqW3IQkka0nNNjHhAArQOIvIzZmEqAMR0Q/edit#
I am sorry we did not announce it team wide.

Reply:
Should I then make this a boolean field? or should we leave it as a varchar for potential future statuses?

Reply:
I think leave it as a varchar makes it both extensible, like you said, and make the serializer easy. Notice how the API returns varchar type `active` vs `inactive`